helpers/setup_next_run.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-### /glade/u/apps/opt/conda/envs/npl/bin/python ??
+
```

Comment:
right... seems like you've responded some of my comments already? 😕 I probably shouldn't go through reviewing each commit one at a time ;)

Reply:
Yes I think I adressed all these in another PR , but dont really understand why they are not implemented here.